lib/scripts/index.js

```diff
@@ -17,8 +17,9 @@ $(() => {
       url = $form.attr( 'action' )
       var posting = $.post( url, { name: $('#newFoodName').val(), calories: $('#newFoodCalories').val() })
       posting.done(function(data) {
-        alert('success')
+        alert('Success! Your food was created.')
         console.log(data)
+        $('#newFood').trigger("reset")
```

Comment:
@somedayrainbows @NessEFC is it necessary to chain two done callbacks here? could you do it all in one?